app/views/admin/order_cycles/checkout_options.html.haml

```diff
@@ -0,0 +1,82 @@
+= render partial: "/admin/order_cycles/order_cycle_top_buttons"
+
+- content_for :page_title do
+  = t :edit_order_cycle
+
+- payment_methods = order_cycle_distributors_payment_methods(@order_cycle)
+- shipping_methods = order_cycle_distributors_shipping_methods(@order_cycle)
+
+= form_for [main_app, :admin, @order_cycle], html: { class: "order_cycle" } do |f|
+
+  = render 'wizard_progress'
+
+  %fieldset.no-border-bottom
+    %legend{ align: 'center'}= t('.checkout_options')
+
+  = hidden_field_tag "order_cycle[selected_shipping_method_ids][]", ""
+
+  .row
+    .three.columns
+      &nbsp;
+    .ten.columns
+      %table.checkout-options
+        %thead
+          %tr
+            %th{ colspan: 2 }= t('.shipping_methods')
+          %tr{ "data-controller": "select-all" }
+            %td.text-center
+              %label
+                = check_box_tag "bla", nil, shipping_methods == @order_cycle.shipping_methods, { "data-action": "change->select-all#toggleAll", "data-select-all-target": "all" }
```

Comment:
I think that `shipping_methods == @order_cycle.shipping_methods` relies on the order if it's an array. Maybe ActiveRecord is clever enough to compare here. I'm not sure. Would be good to double-check.

Reply:
I added a spec and fixed it.